starcheck/src/starcheck.pl

```diff
@@ -346,22 +250,6 @@
     warning("uname == SunOS; starcheck is only approved on Linux \n");
```

Comment:
I can just cut up a few more lines from the Dark cal checker and remove this too.